examples/assets/prefab/sphere.ron

```diff
@@ -0,0 +1,50 @@
+#![enable(implicit_some)]
+Prefab (
+    entities: [
+        (
+            data: (
```

Comment:
IMO we need field names here, the `None` values are not self-documenting otherwise (which I think is a useful thing).

Reply:
This is already fixed in a followup PR.

Reply:
With the followup PR, you don't need to add the None fields at all, you only have to add the fields that you actually need.